src/tests/support.c

```diff
@@ -405,3 +410,33 @@ set_default_rsa_key_desc(rnp_keygen_desc_t *key_desc, pgp_hash_alg_t hashalg)
     key_desc->crypto.rsa.modulus_bit_len = 1024;
     key_desc->crypto.hash_alg = hashalg;
 }
+
+bool
+get_random(uint8_t *data, size_t len)
+{
+    bool ret = false;
+
+    static bool is_rng_initialized = false;
```

Comment:
I think it's better to just make this a global up there next to `global_rng`, so that we're not hiding non-local state here (plus we have a rule in the dev guide about non-const static locals).